README.md

```diff
@@ -19,11 +19,11 @@ See in particular the [fscloud module](./modules/fscloud/) that enables creating
     * [cbr-zone-module](./modules/cbr-zone-module)
     * [fscloud](./modules/fscloud)
 * [Examples](./examples)
-    * [CBR multi service profile](./examples/multi-service-profile)
-    * [Multi resource rule example](./examples/multi-resource-rule)
-    * [Multi-zone example](./examples/multizone-rule)
-    * [Pre-wired CBR configuration for FS Cloud example](./examples/fscloud)
-    * [Zone example](./examples/zone)
+    * [CBR Multi Service Profile](./examples/multi-service-profile)
+    * [Multi-Resource Rule Example](./examples/multi-resource-rule)
+    * [Multi-zone Example](./examples/multizone-rule)
+    * [Pre-wired CBR Configuration for FS Cloud Example](./examples/fscloud)
+    * [Zone Example](./examples/zone)
```

Comment:
these should all go back to the original capitalization after the files are updated and the pre-commit hook runs.